src/rdiff_backup/SetConnections.py

```diff
@@ -517,25 +519,30 @@ def _test_connection(conn_number, rp):
     # the function doesn't use the log functions because it might not have
     # an error or log file to use.
     print("Testing server started by: ", __conn_remote_cmds[conn_number])
-    conn = Globals.connections[conn_number]
+    conn = specifics.connections[conn_number]
     if conn is None:
-        sys.stderr.write("- Connection failed, server tests skipped\n")
+        log.Log("Connection failed, server tests skipped", log.ERROR)
```

Comment:
Any reason to use `log.Log()` instead of `sys.stderr.write()` ? 

I wonder why we have a mixture with `print()` too...

Reply:
Only consistency, I want to have everything going through the log framework. And the `print`'s were an oversight from my side, fixed in the just added commit. Thanks.